### ISSUE TITLE
ACCESSIBILITY: Fix color contrast in navbar

### ISSUE THREAD

**Issue by colinxfleming** [DCAFEngineering/dcaf_case_management#1754]
Thanks for creating an issue! Please fill out this form so we can be sure to have all the information we need, and to minimize back and forth.

* **What are we trying to do?**
To make our app accessible we want to make sure colors contrast enough. To that end we should fix these color contrast issues on the main navbar:

* `<li class="navbar-text-alt">Your current line: DC</li>`: Element has insufficient color contrast of 3.52 (foreground color: #828282- background color: #f5f5f5- font size: 12.8pt- font weight: normal). Expected contrast ratio of 4.5:1. *Please change the foreground color so it's darker.*

We can fix these by making the text hexcode darker using the color contrast tool.

* **What feature or behavior is this required for?**

Accessibility!

* **How could we solve this issue? (Not knowing is okay!)**

Make the text color darker in that class! I think that is in `application.css.scss` but don't quote me on that.

* **Anything else?**
You can use this to pick new colors for text: https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=webdriverjs
Please pick something where all four squares pass!

Stems from #1704

**Comment by colinxfleming** [comment]
This is in app/views/layouts/_navbar_links.html.erb

**Comment by colinxfleming** [comment]
(Please leave for tech rebalanced volunteers!)
